src/App/Fossa/FossaAPIV1.hs

```diff
@@ -283,10 +302,22 @@ instance FromJSON IssueType where
     "outdated_dependency" -> pure IssueOutdatedDependency
     other -> pure (IssueOther other)
 
+instance ToJSON IssueType where
```

Comment:
wish we were using a bidirectional parser 🤔 . lots of room for error in defining FromJSON/ToJSON separately

Reply:
Can we unit test for this so that those potential errors get caught?